src/shared/models/game/GameState.es6

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+//
+// GameState
+// Created by dpekar on 10/7/15.
+//
+
+const BaseModel = require('../BaseModel')
+const GameSetup = require('./GameSetup')
+// const BaseError = require('../error/BaseError')
+// const Player = require('./Player')
+// const Market = require('./Market')
+// const Tile = require ('./Tile')
+const _ = require('underscore')
+
+class GameState extends BaseModel {
+
+    constructor(users) {
+        super()
+        this.init(users)
+    }
+
+    init(users) {
+        this.setupPlayers(users)
+        this.setupTilePiles()
+        this.setupStartingTilesForPlayers()        
+        this.setupMarket()        
+        return this
+    }
+
+    getPlayers() {return this.get('players')}
+    getTilePiles() {return this.get('tilePiles')}
+    getMarket() {return this.get('market')}
+
+
+    placeTile(tile, player) {
+        // remove a tile from the market or basic market
```

Comment:
as you think about these steps.  we'll probably want to make each step something we can call separately.  i can see, in the client, us wanting to play out each step one at a time